libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.m

```diff
@@ -44,6 +44,7 @@
 static NSString * const kEventStoresDirectory = @"event_stores";
 static NSString * const kEventStoreEncryptionKeyLabel = @"com.salesforce.eventStore.encryptionKey";
 static NSString * const kAnalyticsOnOffKey = @"ailtn_enabled";
+static NSString * const kSFAppFeatureAiltnEnabled = @"AI";
```

Comment:
Well, we _could_ have the former.  I.e. they could be denormalized, since they don't otherwise compromise referential integrity—they're just strings, at the end of the day.